crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -17,9 +13,14 @@ use std::{
     time::Duration,
 };
 use tempfile::TempDir;
+use tokio::task;
 
 use crate::config::{Config, SandboxConfig, SANDBOX_OCTEZ_SMART_ROLLUP_PORT};
 
+const SANDBOX_JSTZ_NODE_ADDR: &str = "127.0.0.1";
+const SANDBOX_JSTZ_NODE_PORT: u16 = 8933;
+const SANDBOX_OCTEZ_SMART_ROLLUP_ADDR: &str = "127.0.0.1";
```

Comment:
Now we have it defined it two places which is not ideal. Ideally there should be a shared library for these constants.

Reply:
They don't necessarily have to be the same -- the node has defaults for it's CLI and we have defaults for the sandbox. 

They're only currently the same because the node was designed with the sandbox in mind